src/models/wallet/summary.ts

```diff
@@ -75,7 +68,7 @@ const walletSummary: WalletSummaryModel = {
   immature: 0,
   unconfirmed: 0,
   locked: 0,
-  updateSummaryInterval: 1000,
+  updateSummaryInterval: 3000,
```

Comment:
Ah, I didn't realize this was so low. I thought we had it at 10 seconds before. I must have missed when we changed that. 

This call is quite a bit heavier than the node summary one, since it actually obtains locks and does a bit of processing some times. 3 seconds is probably fine, but I agree, 1 second was probably too often.

Reply:
I think 5s... we can test it and change this after.